gl-pvr.c

```diff
@@ -96,8 +96,8 @@ static inline void pvr_hdr_submit(const GLuint *src) {
 }
 #else
 
-#define pvr_list_submit(src, n) sq_cpy(TA_SQ_ADDR, (src), ((n) << 5))
-#define pvr_hdr_submit(src) sq_cpy(TA_SQ_ADDR, (src), 32)
```

Comment:
For the same reason as the above, go ahead and remove the TA_SQ_ADDR define. It was only used in the above and in these and sets manual values for sq use, which isn't needed anymore.